app/src/main/java/com/codesoom/assignment/common/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.common.exceptions;
+
+public class UserNotFoundException extends RuntimeException {
+
+    public UserNotFoundException() {
+        super("회원을 찾을 수 없습니다.");
+    }
+
+    public UserNotFoundException(String message) {
+        super(message);
+    }
```

Comment:
아래 생성자가 주 생성자라면, 위의 생성자는 부 생성자가 될 것이고 그러면 사실 다음과 같이 되는게 좋겠네요.

```suggestion
    public UserNotFoundException() {
        this("회원을 찾을 수 없습니다.");
    }

    public UserNotFoundException(String message) {
        super(message);
    }
```